### ISSUE TITLE
deltares data argument in documentation

### ISSUE THREAD

**Issue by shartgring** [Deltares/hydromt_wflow#235]
### HydroMT-Wflow version checks

- [X] I have checked that the issue still exists on the latest versions of the docs on `main` [here](https://github.com/Deltares/hydromt_wflow)

### Kind of issue

Docs are wrong

### Location of the documentation

https://deltares.github.io/hydromt_wflow/latest/user_guide/wflow_build.html

Another options for Deltares users is to select the deltares-data library (requires access to the Deltares P-drive). In the command lines examples below, this is done by adding either –dd or –deltares-data to the build / update command line.

### Documentation problem

-dd is given as option, should be --dd

Using -dd will result in hydromt thinking that a catalog with name d is added resulting in the following error:

`ValueError: Catalog with name "d" not found in predefined catalogs`

### Suggested fix for documentation

Replace -dd with --dd in documentation

**Comment by hboisgon** [comment]
Good catch! Actually --dd will be deprecated soon so better also already remove this option from our docs.
